### PR TITLE
Adds CLI options for setting path to binary for selftests

### PR DESCRIPTION
**Pull request by tobie**
This allows setting the path to the binary from tox like so:

    $ tox -- --binary-path=/path/to/FirefoxNightly

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/6744)

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260699984)

# Firefox (nightly) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260699985)

# Sauce (safari) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260699992)

# Chrome (unstable) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/260699993)

# Sauce (MicrosoftEdge) #

File: resources/test/conftest.py [hunk 1]
```diff
@@ -13,12 +13,15 @@
...
+def pytest_addoption(parser):
+    parser.addoption("--binary-path", action="store", default=None, help="path to browser binary")
+
 def pytest_collect_file(path, parent):
     if path.ext.lower() == '.html':
         return HTMLItem(str(path), parent)
 
 def pytest_configure(config):
-    config.driver = webdriver.Firefox()
+    config.driver = webdriver.Firefox(firefox_binary=config.getoption("--binary-path"))
```
Comment (jgraham) [inline resources/test/conftest.py]: Can we use just `--binary` since that's what `wpt run` uses.

**Review by jgraham — CHANGES_REQUESTED** [review]

File: resources/test/conftest.py (hunk 1 above)
Comment (tobie) [inline resources/test/conftest.py]: Sure. Fixed.

File: resources/test/README.md [hunk 2]
```diff
@@ -16,6 +16,12 @@ by executing the following command from this directory:
 
     tox
 
+Currently, the tests should be run with FirefoxNightly.
```
Comment (jgraham) [inline resources/test/README.md]: Space after Firefox

**Review by jgraham — APPROVED** [review]

File: resources/test/README.md (hunk 2 above)
Comment (tobie) [inline resources/test/README.md]: Done.

**Pull request merged**